x-pack/legacy/plugins/siem/server/lib/case/saved_object_mappings_temp.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+/* eslint-disable @typescript-eslint/no-empty-interface */
+/* eslint-disable @typescript-eslint/camelcase */
+import { ElasticsearchMappingOf } from '../../utils/typed_elasticsearch_mappings';
+import {
+  NewCaseFormatted,
+  NewCommentFormatted,
+} from '../../../../../../plugins/case/server/routes/api/types';
+
+// Temporary file to write mappings for case
+// while Saved Object Mappings API is programmed for the NP
+// See: https://github.com/elastic/kibana/issues/50309
+
+export const caseSavedObjectType = 'case-workflow';
+export const caseCommentSavedObjectType = 'case-workflow-comment';
+
+export const caseSavedObjectMappings: {
+  [caseSavedObjectType]: ElasticsearchMappingOf<NewCaseFormatted>;
+} = {
+  [caseSavedObjectType]: {
+    properties: {
+      assignees: {
+        properties: {
+          username: {
+            type: 'keyword',
+          },
+          full_name: {
+            type: 'keyword',
+          },
+        },
+      },
+      creation_date: {
```

Comment:
OCD's comment from my part, sorry! ;) I am used to seeing `created`, `createdBy`, `updated` and `updatedBy` has a more generic naming

Reply:
I'm going to remove `last_edit_date` and just use the saved object's `updated_at`. And I will change to `created_at` to match the saved object's convention

Reply:
they might add a created_at at one point so it might be confusing but we can always deal with that later on. Do you have a `createdBy` and `updatedBy`?

Reply:
`reporter`, ill change to `created_by` (snake case to match convention). I'm designing this after a github issue, and if you notice when you update someone else's issue, it does not notate your name at all. think we should be tracking it?

Reply:
I still think that we should do it because a user would like to know who did what when on the case